src/HelpPage.tsx

```diff
@@ -217,11 +217,15 @@ function HelpPage() {
             External links and virtual datasets in HDF5 files are not supported.
             While you should see an explicit error for external links, it won't
             be the case for virtual datasets, which will appear filled with
-            zeros.
-          </li>
-          <li>
-            Datasets compressed with external filters (e.g. bitshuffle) can be
-            inspected but not visualized.
+            zeros (or with the dataset's{' '}
+            <a
+              href="https://docs.hdfgroup.org/hdf5/develop/group___d_c_p_l.html#title28"
+              target="_blank"
+              rel="noreferrer"
+            >
+              fill value
+            </a>
+            , if set).
```

Comment:
Since we've recently configured HDF5 lib to throw errors, I thought I'd double checked the behaviour for virtual datasets. No errors still, but in the example file from h5py, the virtual dataset has a fill value of -5 instead of the default of 0. I thought I'd clarify this here.

Reply:
Good to know :+1: